draft-ietf-quic-transport.md

```diff
@@ -5415,14 +5426,14 @@ active_connection_id_limit (0x0e):
 
 initial_source_connection_id (0x0f):
 
-: The value that the endpoint included in the Source Connection ID field of the
-  first Initial packet it sends for the connection; see {{cid-auth}}.
+: This is the value that the endpoint included in the Source Connection ID field
```

Comment:
The use of "This is" here (and below) seems to be arbitrary and doesn't seem uniform, but I could be missing something. What is the logic here?